metaflow/cmd/make_wrapper.py

```diff
@@ -28,6 +28,27 @@ def find_makefile():
         if makefile_candidate.is_file():
             return makefile_candidate
 
+    # 4) When developing, mMtaflow might be installed with --editable, which means the devtools will not be located within site-packages.
+    # We read the actual location from package metadata in this case, but only do this heavier operation if the above lookups fail.
+    try:
+        import json
+        from importlib.metadata import Distribution
```

Comment:
for py < 3.10, you would want to do - from metaflow._vendor.importlib_metadata ..